### ISSUE TITLE
type 1: Feed Forward Network (1 layer)

### ISSUE THREAD

**Issue by anupamme** [anupamme/keras-triplet-loss-mnist#1]
The way we do this is create an embedding layer followed by a dense layer:

tf.keras.layers.Dense(256, activation=None)

It has an implementation in the fuction **get_model_FF**
